### PR TITLE
Patchinfo tests

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/app/views/webui/patchinfo/show.html.haml [hunk 1]
```diff
@@ -41,7 +41,10 @@
       - if @issues.present?
         %ul
           - @issues.each do |issue|
-            = issue_link(issue)
+            - if issue[0] == "CVE"
+              = content_tag(:li, link_to((issue[1]).to_s, issue[2]) + ": #{issue[3]}")
```
Comment (DavidKang) [inline src/api/app/views/webui/patchinfo/show.html.haml]: Why content_tag? and not something like ` %li= link_to((issue[1]).to_s, issue[2]) + ": #{issue[3]}"`

File: src/api/app/views/webui/patchinfo/show.html.haml [hunk 2]
```diff
@@ -41,7 +41,10 @@
...
+            - else
+              = content_tag(:li, link_to("#{issue[0]}##{issue[1]}", issue[2]) + ": #{issue[3]}")
```
Comment (DavidKang) [inline src/api/app/views/webui/patchinfo/show.html.haml]: same here

**Comment by DavidKang** [comment]
Nice work!

File: src/api/app/views/webui/patchinfo/show.html.haml (hunk 1 above)
Comment (bgeuken) [inline src/api/app/views/webui/patchinfo/show.html.haml]: Good point. Didn't think of that

**Comment by bgeuken** [comment]
@DavidKang Changed it now

**Pull request merged**